crates/l2/contracts/deployer.rs

```diff
@@ -556,8 +612,12 @@ mod test {
             }
         }
 
-        download_contract_deps(Path::new("contracts"));
-        compile_contracts(Path::new("contracts"));
+        if download_contract_deps(Path::new("contracts")).is_err() {
+            panic!()
+        };
+        if compile_contracts(Path::new("contracts")).is_err() {
+            panic!()
+        };
```

Comment:
We should set a comment inside the panics

Reply:
Done in [bc2e6cc](https://github.com/lambdaclass/ethrex/pull/1290/commits/bc2e6cc1da537cc628ee0699f48ac78524898989)